### ISSUE TITLE
I have a question about pointdan accuracy.

### ISSUE THREAD

**Issue by brandon9838** [IdanAchituve/DefRec_and_PCM#1]
I noticed the reported accuracy of pointdan is different from the original paper.
http://papers.nips.cc/paper/8940-pointdan-a-multi-scale-3d-domain-adaption-network-for-point-cloud-representation.pdf
Is it due to aligning the upward direction of 3 datasets, or there are other reasons?
How can I reproduce the experiment on pointdan?
Thanks for replying!

**Comment by idanachi** [comment]
Hi Brandon,
The results are indeed different. There are several factors causing this
discrepancy. I think the following are the main reasons:
1. As you stated we did an alignment of the shapes with the upwards
direction. AFAIK they regarded it as part of the domain shift.
2. PointDAN didn't split the train set to train part and validation part.
We did and applied early stopping according to the validation *source*
performance (i.e., assuming no target labels at all during the training
process).
3. Our understanding is that PointDAN did early stopping based on the
target test set.

We used their published code with the setting depicted in our paper in
order to do a fair comparison.

Cheers,
Idan
